hack/build-antrea-windows-all.sh

```diff
@@ -69,7 +69,8 @@ pushd "$THIS_DIR/.." > /dev/null
 NANOSERVER_VERSION=$(head -n 1 build/images/deps/nanoserver-version)
 CNI_BINARIES_VERSION=$(head -n 1 build/images/deps/cni-binaries-version)
 GO_VERSION=$(head -n 1 build/images/deps/go-version)
-WIN_BUILD_TAG=$(echo $GO_VERSION $CNI_BINARIES_VERSION $NANOSERVER_VERSION| md5sum| head -c 10)
+WIN_OVS_VERSION=$(head -n 1 build/images/deps/ovs-version-windows)
+WIN_BUILD_TAG=$(echo $GO_VERSION $CNI_BINARIES_VERSION $NANOSERVER_VERSION | md5sum| head -c 10)
```

Comment:
If you want to involve ovs build in base image, we need to add WIN_OVS_VERSION in WIN_BUILD_TAG.

Reply:
@XinShuYang , if we add WIN_OVS_VERSION to WIN_BUILD_TAG, build windows docker image check fails , plz check error here from previous quote,
> https://github.com/antrea-io/antrea/actions/runs/4913939105/jobs/8774697360?pr=4936

So, we will have to upload new windows base image with new tag(which includes ovs) and then build antrea windows dockr image .
 If we can merge this PR after adding WIN_OVS_VERSION to WIN_BUILD_TAG with exception of building windows docker image, and only build windows docker image after merge then we can add the ovs tag.

Reply:
@XinShuYang ,
Because it is trying to pull the new base windows image with new tag(with ovs) which is currently not available, CI build antrea windows image test will fail) , we will have to push new image to registry before this CI test can pass.

Step 3/21 : FROM antrea/base-windows:${WIN_BUILD_TAG} as antrea-build-windows
manifest for antrea/base-windows:0af6780141 not found: manifest unknown: manifest unknown